frontend/src/metabase/dashboard/components/AddActionSidebar/utils.ts

```diff
@@ -0,0 +1,4 @@
+export const hasLinkDestination = (clickBehavior: any) =>
+  clickBehavior.type === "link" &&
+  clickBehavior.linkType &&
+  (clickBehavior.targetId || clickBehavior.linkTemplate);
```

Comment:
Think it should better live in `click-behavior` lib

Reply:
Or we can just use [`clickBehaviorIsValid` that does the same thing](https://github.com/metabase/metabase/blob/875137b6bce38ea386cf86dbf3a65c3aac693c76/frontend/src/metabase/lib/click-behavior.js#L264)